funnel/templates/macros.html.jinja2

```diff
@@ -120,7 +121,7 @@
           {%- if transition_form %}
             <div>
               <p class="mui--text-body2 mui--text-light zero-bottom-margin mui--text-uppercase">Status</p>
-              <p class="mui--text-subhead">{{ project.old_state.label.title }}</p>
+              <p class="mui--text-subhead">{{ project.state.label.title }}</p>
               <p class="mui--text-body2 mui--text-light mui--text-uppercase zero-bottom-margin">Change status</p>
               <form action="{{ project.url_for('transition') }}" method="post" class="form-inline">
                 {{ transition_form.hidden_tag() }}
```

Comment:
This section will change. Event managers will only have two transitions in the lifetime of the event:

1. Draft to Published
2. Published to Withdrawn

The first one should be very prominent. The second one is dangerous, so it should be hidden away in a settings page.